src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -96,68 +119,93 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
                         notifications.ShowError(error?.Message ?? ex.Message);
                         return Observable.Empty<IPullRequestModel>();
                     }));
+            isExecuting = CreatePullRequest.IsExecuting.ToProperty(this, x => x.IsExecuting);
```

Comment:
`IsBusy` reflects a combination of multiple states that signify the vm is doing something and so data should not be externally changed until that something is finished (when loading branches, initializing, executing the creation, etc)

I really don't like having to have this `IsExecuting` property just to listen to the changes in the observable of the button, but right now it's the best way to combine it with the rest of the property changes. I really want to find a cleaner way of doing this next week.